### DISCUSSION TITLE
Bindings error

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#9028] [Help]
Does anybody know what I might be doing wrong here?

BTW: when I uses Test instead of {{table}} in the UPDATE query, everything works fine ...

![Screenshot 2022-12-13 at 13 39 57](https://user-images.githubusercontent.com/4005405/207320725-40230d52-cb69-401b-935a-1b5add29c0a5.png)

**Comment by AlejandroFNadal** [comment]
Hello :)

Have you tried with `Test`, 'Test',  or "Test" in the Default field instead of Test?

**Reply by ecxs** [reply]
Hi @AlejandroFNadal , yes I have ... these also return the same error ... Query Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''\"Test\"' SET status_sql = '1' WHERE id_sql = 1' at line 1

**Comment by melohagan** [comment]
Hey @ecxs 

Don't use **table** as a binding name. Table is a keyword in SQL which may be causing the issue.

**Reply by melohagan** [reply]
Ah I'm just noticing it is wrapping the table name in double quotes. That's probably the issue.

**Reply by ecxs** [reply]
@melohagan tried with a different binding name, but unfortunately, still the same problem ...

![Screenshot 2022-12-21 at 15 20 53](https://user-images.githubusercontent.com/4005405/208927006-cfc9132d-5858-4a2f-9fcb-32a8aabd7596.png)

**Reply by ecxs** [reply]
@melohagan double quotes is also not the problem ... I tried almost every possible combination ...

**Reply by melohagan** [reply]
It's actually single quotes. What I mean is that this is probably a bug/limitation. The error message shows `'Test' SET ...` but it probably needs to be `Test SET ...`. It may not currently be possible to use a dynamic table name.

**Reply by ecxs** [reply]
@melohagan thanks for letting me know that I'm not getting crazy ;-)
